strings.txt

```diff
@@ -8414,21 +8414,27 @@ SETUP_ENABLE_STATISTICS
 	SV	Aktivera biblioteksstatistik
 	ZH_CN	启用库存统计
 
-SETUP_BUFFEREDHTTP
-	DE	Online Datenströme zwischenspeichern
-	EN	Cache online streams to disk
+SETUP_ENHANCEDHTTP
+	EN	Streaming mode for HTTP(S)
+	FR	Mode de streaming HTTP(S)
 
-SETUP_BUFFEREDHTTP_DESC
+SETUP_ENHANCEDHTTP_DESC
 	DE	Logitech Media Server kann HTTP(S) Datenströme mit definierter Länge (Podcasts, einzelne Lieder) im Dateisystem zwischenspeichern. Sie würden wieder gelöscht, sobald der Strom zu Ende gespielt ist. Dies erhöht die Zuverlässigkeit mit einigen Anbietern, kann aber das Dateisystem zusätzlich belasten (z.B. wenn eine SD Karte verwendet wird).
-	EN	Logitech Media Server can cache HTTP(S) streams of defined length (podcasts, single tracks) on disk. The buffer files will be removed again once the stream has been played. This can improve the reliability with some servers. But it also does cause more writing to the file system, which could potentially wear out eg. SD cards.
+	EN	When a HTTP(S) connection to a server fails, Logitech Media Server terminates playback. It can also try to re-open it (Persistent mode) or it can buffer streams of defined length (podcasts, single tracks) on disk files. These will be removed once the stream has been played. This can improve the reliability with some servers that expect tracks to be downloaded, not streamed and thus close long connections. But it also does cause more writing to the file system, which could potentially wear out eg. SD cards. The buffer directory is cache/tmp but can be changed using --tmpdir on the slimserver.pl command line
```

Comment:
Could you please make it clearer what the pros/cons are using one or the other option? And I'd prefer not to mention the `tmpdir` parameter here. I fear this will cause more trouble than actually help people when used by people who don't understand the consequences (eg. `/tmp` in a ramdisk too small to be helpful, causing LMS to crash).

Reply:
Done